components/ErrorSummary/ErrorSummary.module.css

```diff
@@ -0,0 +1,6 @@
+.restricted_summary_box {
```

Comment:
I am being picky here, but we should use css modules or override the style in all, not both 😅